velox/dwio/parquet/common/BloomFilter.h

```diff
@@ -0,0 +1,360 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Adapted from Apache Arrow.
+
+#pragma once
+
+#include "Hasher.h"
+#include "velox/common/base/BitUtil.h"
+#include "velox/dwio/common/BufferedInput.h"
+#include "velox/dwio/common/OutputStream.h"
+#include "velox/dwio/parquet/thrift/ParquetThriftTypes.h"
```

Comment:
We cannot have thrift dependency in core modules of velox.  Since this class is not written in a way that is efficient enough to be reused throughout the code base, can we move it into `dwio/parquet`?

Reply:
@Yuhta. This file is in `dwio/parquet/common`, not in core velox. When we implement writer, won't it be easy if this remains in common package?

Reply:
Sorry I misread the path, it is good then